pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -778,6 +800,7 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 				StartupStrategy:          cr.Status.Applied.CommonConfig.StartupStrategy,
 				DecisionsOnly:            serverSet.DecisionsOnly,
 				KieExecutorMDBMaxSession: serverSet.KieExecutorMDBMaxSession,
+				SecretAdminCredentials:   cr.Status.Applied.CommonConfig.SecretAdminCredentials,
```

Comment:
is this secret really needed here?

Reply:
Yes otherwise the yaml in the loop of the []servertemplate can't access to it

Reply:
Fixed